### PR TITLE
Convert CornieAppTheme and cornieTheme from JS to TS

### PR DESCRIPTION
**Pull request by wariofdz**
### Changed:

- Transformed `CornieAppTheme` component from JSX to TSX.
- Transformed `cornieTheme` configuration file from JS to TS.

### CODE REVIEW

File: packages/frontend/web-ui/src/app/theme/cornieTheme.ts [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
-const corniePalette = {
+interface CorniePaletteOptions extends PaletteOptions {
+  primary?: SimplePaletteColorOptions;
+  secondary?: SimplePaletteColorOptions;
+}
+
+const corniePalette: CorniePaletteOptions = {
   primary: {
-    white: '#fff',
+    //white: '#fff',
```
Comment (notaphplover) [inline packages/frontend/web-ui/src/app/theme/cornieTheme.ts:15]: Avoid comment in favor of removing the line

File: packages/frontend/web-ui/src/app/theme/cornieTheme.ts [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+  PaletteOptions,
+  SimplePaletteColorOptions,
+  createTheme,
+} from '@mui/material/styles';
 
-/** @type {import("@mui/material/styles").PaletteOptions } */
-const corniePalette = {
+interface CorniePaletteOptions extends PaletteOptions {
+  primary?: SimplePaletteColorOptions;
+  secondary?: SimplePaletteColorOptions;
```
Comment (notaphplover) [inline packages/frontend/web-ui/src/app/theme/cornieTheme.ts:10]: Why are these properties optional?

**Review by notaphplover — CHANGES_REQUESTED** [review]
Nice, I just added some comments

File: packages/frontend/web-ui/src/app/theme/cornieTheme.ts (hunk 2 above)
Comment (wariofdz) [reply packages/frontend/web-ui/src/app/theme/cornieTheme.ts:10]: I've changed this!

File: packages/frontend/web-ui/src/app/theme/cornieTheme.ts (hunk 1 above)
Comment (wariofdz) [reply packages/frontend/web-ui/src/app/theme/cornieTheme.ts:15]: Deleted

**Review by notaphplover — APPROVED** [review]

**Pull request merged**